packages/outstatic/src/types/index.ts

```diff
@@ -46,3 +46,7 @@ export type Session = {
 export type Collection = {
   name: string
 }
+
+export type DeepNonNullable<T> = {
```

Comment:
Recursively applies the `NonNullable<T>`. Open to other ways to do this, but I couldn't think of any that are proper TypeScript